aptos-move/aptos-gas-calibration/README.md

```diff
@@ -56,23 +56,73 @@ public entry fun calibrate_addition(_x: u64, _y: u64) {}
 
 If the Calibration Function is expected to error, please denote it with the postfix `_should_error`.
 
+```
+//// VALID
+public entry fun calibrate_my_test_should_error() {}
+
+public entry fun calibrate_should_error() {}
+
+//// INVALID
+public entry fun should_error_calibrate() {}
+
+public entry fun calibrate_test_error() {}
+
+public entry fun calibrate_test_should_error_() {}
+```
+
 ## Usage
 
 ```bash
-cargo run -- --help 
+cargo run --release -- --help 
 Automated Gas Calibration to calibrate Move bytecode and Native Functions
 
 Usage: aptos-gas-calibration [OPTIONS]
 
 Options:
-  -p, --pattern <PATTERN>        Specific tests to run that match a pattern [default: ]
-  -i, --iterations <ITERATIONS>  Number of iterations to run each Calibration Function [default: 20]
-  -h, --help                     Print help
+  -p, --pattern <PATTERN>                         Specific tests to run that match a pattern [default: ""]
+  -i, --iterations <ITERATIONS>                   Number of iterations to run each Calibration Function [default: 20]
+  -m, --max_execution_time <MAX_EXECUTION_TIME>   Maximum execution time in milliseconds [default: 300]
+  -h, --help                                      Print help
 ```
 
 ## Examples
 
-There are examples of how to write Calibration Functions under `/samples_ir` and `/samples`. There will be more examples in the future as more Users write Move Samples and add it to the calibration set.
+There are examples of how to write Calibration Functions under `/samples_ir` and `/samples`. There will be more examples in the future as more Users write Move Samples and add it to the calibration set. 
+
+Here is an example written in the Move source language:
+```Move
+public fun calibrate_blake2b_256_impl(n: u64) {
+    let i = 0;
+    let msg = b"abcdefghijkl"
+    while i < n {
+        // This is what I want to calibrate:
+        aptos_hash::blake2b_256(msg);
+        aptos_hash::blake2b_256(msg);
+        aptos_hash::blake2b_256(msg);
+        aptos_hash::blake2b_256(msg);
+        aptos_hash::blake2b_256(msg);
+        aptos_hash::blake2b_256(msg);
+        aptos_hash::blake2b_256(msg);
+        aptos_hash::blake2b_256(msg);
+        aptos_hash::blake2b_256(msg);
+        aptos_hash::blake2b_256(msg);        
+        i += 1;
+    }
+}
```

Comment:
This example is confusing. The parameter `n` is supposed to control the length of the vector used but it's not.

Reply:
Maybe a better name is to call `n` the `num_iterations` instead for the while-loop

Reply:
Oh my bad I misread